app/views/layouts/_google_tag_manager.html.erb

```diff
@@ -0,0 +1,7 @@
+<% if ENV['GOOGLE_TAG_MANAGER_TRACKING_ID'].present? %>
```

Comment:
Lines 1 and 7 are redundant - the conditional has been moved to the `render` call in the main application layout template.